README.md

```diff
@@ -8,6 +8,15 @@ For changes, see the [Changelog](CHANGELOG.md).
 
 ## Development Environment Setup
 
+To install the classes in your local Python env, run:
+
+```shell
+cd stac_fastapi/elasticsearch
+pip install -e .[dev]
```

Comment:
this command was missing the `-e`, so changes to the code weren't reflected in the IDE.